src/typings/formatters.ts

```diff
@@ -74,7 +74,7 @@ type ParsedFieldName = {
  * Parsed field placeholder from a YScope format string.
  */
 type YscopeFieldPlaceholder = {
-    parsedFieldName: ParsedFieldName;
+    parsedFieldName: ParsedKey;
```

Comment:
just curious, how do we differentiate between a `parsedKey` and a `parsedFieldName` after this type renaming?

Reply:
We discussed offline:

The concept of "field names" are usually referenced in the context of field formatters / formatter strings. The docs also references this part of the specifier as `<field-name>`. For field formatters, it is fine to leave the name as-is. We can rename the field in a future refactoring PR if the name also changes in the docs.